quadratic-core/src/controller/export.rs

```
@@ -9,7 +9,10 @@ impl GridController {
     ///
     /// Returns a [`String`].
     pub fn export_csv_selection(&self, sheet_id: SheetId, selection: &Rect) -> Result<String> {
-        let sheet = self.sheet(sheet_id);
+        // todo: handle this better
+        let Some(sheet) = self.try_sheet(sheet_id) else {
+            return Ok("".to_string());
```

Since your return signature is a result, would you want to return an error here?

---

No. A sheet not found is no longer an error as it can happen during multiplayer transactions.